tests/data-processing.feature

```diff
@@ -0,0 +1,28 @@
+# This file is part of REANA.
```

Comment:
You can amend the leading comments as discussed in CMS H4L and rebase against latest master (where the CI issue has been fixed).